common/install_nvidiagpudriver.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -ex
+
+# Install Cuda
+wget https://developer.download.nvidia.com/compute/cuda/11.0.3/local_installers/cuda_11.0.3_450.51.06_linux.run
+chmod +x cuda_11.0.3_450.51.06_linux.run
+sudo ./cuda_11.0.3_450.51.06_linux.run --silent
+echo 'export PATH=$PATH:/usr/local/cuda/bin' | sudo tee -a /etc/bash.bashrc
+echo 'export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/local/cuda/lib64' | sudo tee -a /etc/bash.bashrc
+
+# Install DCGM
```

Comment:
We can delete these commented lines (they wont work anyway for CentOS)

Reply:
Done